doc/exception_interrupts.rst

```diff
@@ -15,6 +15,8 @@ Ibex supports interrupts, exceptions on illegal instructions.
 +------------+-----------------------------+
 | **0x84**   | Illegal Instruction         |
 +------------+-----------------------------+
+| **0x8C**   | LSU Error                   |
```

Comment:
I know this is already closed, but why is the sort order funny here? 0x84 0x8c 0x88?

Reply:
Ui, thanks for catching that @sjgitty . It is my fault, sorry. I will fix it in the next PR.